test/lib/recordevents/event_info_store.go

```diff
@@ -314,6 +318,15 @@ func (ei *EventInfoStore) waitAtLeastNMatch(f EventInfoMatcher, min int) ([]Even
 	return matchRet, internalErr
 }
 
+// This function prints the place where the assert was invoked
+func (ei *EventInfoStore) logAssertCompleted() {
```

Comment:
The `go test` way of doing this is to call `t.Helper()` in the assertion function/method. Why aren't you doing that?

Reply:
I wasn't aware of that :smile: Fixed